sources/packages/web/src/constants/award-constants.ts

```diff
@@ -53,12 +53,6 @@ export const AWARDS: AwardDetail[] = [
     description: "Canada Student Grant for Full-time Studies",
     offeringIntensity: OfferingIntensity.fullTime,
   },
-  {
-    awardType: FullTimeAwardTypes.CSGT,
```

Comment:
What is the reason for CSGT being removed? if CSGT is obsolete then it sounds good.

Reply:
That's what the AC says: "Remove CSGT from award breakdown and funding summary page".

Reply:
Yes. I can understand the action, but was trying to understand the reason. I strongly believe that it is obsolete. But never mind as it is not very important.